locales/en-US/strings.properties

```diff
@@ -5,25 +5,35 @@ bookmarked=Bookmarked
 synced=Synced from another device
 open=Open
 topic=Topic
+# LOCALIZATION NOTE: 'bookmark' is an action meant as 'add to bookmarks'
 bookmark=Bookmark
 remove_bookmark=Remove Bookmark
 share=Share
 copy_address=Copy Address
-email_link=Email Link...
+email_link=Email Link…
 open_new_window=Open in a New Window
 open_private_window=Open in a Private Window
 dismiss=Dismiss
 delete=Delete from History
 newtab=New Tab
 search_web=Search the Web
+# LOCALIZATION NOTE: {search_term} is a placeholder for what the user has inputed
+# in the search input field: 'Search for ' + search_term + 'with'. The search engine
+# name is displayed as an icon and does not need a translation
+# e.g. 'Search for abc with'
 search_for=Search for {search_term} with
```

Comment:
In particular for these search related strings, we can build off existing localization notes. Also, the original string seems to end with a colon `:` (and that makes it seem more clear the whole string is a phrase as opposed to a partial sentence that looks like it would be concatenated with other words): https://dxr.mozilla.org/mozilla-central/source/browser/locales/en-US/chrome/browser/search.properties#36-40